lib/configuration/configuration.js

```diff
@@ -119,9 +141,18 @@ class Configuration {
 
     // probotContext.config loads config from current repo or from a repo called
     // '.github' in the same organisation as a fallback. It returns the parsed YAML object.
-
+    let config = null
     const configPath = process.env.CONFIG_PATH ? process.env.CONFIG_PATH : 'mergeable.yml'
-    const config = await context.probotContext.config(configPath)
+    if (configCache !== null) {
+      Configuration.updateCache(context)
+      config = configCache.get(`${repo.owner}/${repo.repo}`)
+      if (config === undefined) {
+        config = await context.probotContext.config(configPath)
+        configCache.set(`${repo.owner}/${repo.repo}`, config)
+      }
+    } else {
+      config = await context.probotContext.config(configPath)
+    }
     if (config === null || config === undefined) {
```

Comment:
I know this is old code you didn't introduce -- probably been around since before `loadash` was introduced into the codebase, but I think it's time we update this to use `_.isNull` and `_.isUndefined` for consistency.